src/components/Navbar/ContactModal/ContactModal.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+  Button, Checkbox, FormControl, FormControlLabel, Modal, TextField,
+} from '@material-ui/core';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { useStyles } from './style';
+
+export default function ContactModal({
+  open,
+  setOpen,
+}) {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [nameError, setNameError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [nameErrorMessage, setNameErrorMessage] = useState('');
+  const [emailErrorMessage, setEmailErrorMessage] = useState('');
```

Comment:
what about single userInput object with name nad email properties instead? or general error with properties to divide name error and mail error?